### DISCUSSION TITLE
Row checkbox disable

### DISCUSSION THREAD

**Discussion by shohibexlab** [bosskmk/pluto_grid#491] [Q&A]
Hi there.
I am working on some complex Table UI that I need a lot of modifications in the table. I even modified it for myself to fit the UI requirements. I came up with one tricky task that I need to disable some of the checkboxes depending on the row value.
For example, in the row I have a column stating status of product(available and not available). So depending on this, I need to disable the specific row checkbox. Does it come with this feature out of the box or how can I achieve this?
Appreciate your help! Thank you

**Comment by shohibexlab** [comment]
> Hi there. I am working on some complex Table UI that I need a lot of modifications in the table. I even modified it for myself to fit the UI requirements. I came up with one tricky task that I need to disable some of the checkboxes depending on the row value. For example, in the row I have a column stating status of product(available and not available). So depending on this, I need to disable the specific row checkbox. Does it come with this feature out of the box or how can I achieve this? Appreciate your help! Thank you

FYI, I am using **pluto_grid-2.10.3**

**Comment by bosskmk** [comment]
@shohibexlab 
Currently, there is no function to disable the checkbox.
It seems to be a necessary feature.
I will consider adding a function to disable the checkbox according to certain conditions.

Before that, if you want to implement this function, you must manually add a checkbox with cellRenderer and handle it.

You can add widgets to columns with `PlutoColumn.titleSpan`.
https://weblaze.dev/pluto_grid/build/web/#add-and-remove-column-row

You can add widgets to cells with `PlutoColumn.renderer`.
https://weblaze.dev/pluto_grid/build/web/#feature/cell-renderer

It seems possible to implement these two.
The above two links do not add a checkbox, but you can add a checkbox with similar code.

**Reply by shohibexlab** [reply]
Thank you for you response! It helps me a lot!
